conda-recipe/meta.yaml

```diff
@@ -48,22 +49,21 @@ outputs:
         - pandas <2
         - psutil
         - pyopengl
-        - pyqt 5.12.*
+        - pyqt 5.15.*
         - python-elf
         - pytiff
         - scikit-image
         - scikit-learn
-        # last py37 compatible version
-        # newer packages on CF currently don't reflect that
-        - tifffile >2020.9.22,<=2021.11.2
+        # tiffile >2020.9.22,<=2021.11.2  # [py37]
```

Comment:
hmm yeah, I mean we should definitely document it somewhere, since we still support 3.7. Maybe some extended conda installation notes would be useful. I'd still keep the note in the `environment.yml`, at least.